### DISCUSSION TITLE
Album Sharing with other Photoprism Instance

### DISCUSSION THREAD

**Discussion by N20Visuals** [photoprism/photoprism#3337] [Q&A]
Hi, 

i've been running two separate instances of Photoprism for quite some time now. 
I've read through what is planned with the upcoming Photoprism Plus and found that it will still be recommended to have multiple instances running for "full users" so, each familiy member has their own instance, is that correct? 

if yes, is there a way to share albums across photoprism instances? So, i want to share an album of holiday photos with my girlfriends instance and have it show up in her ui? Additionally, if she then could merge her photos of said holiday with my album, that'd be great... 
maybe there is a way or is one upcoming? thank you for suggestions :)

**Comment by lastzero** [comment]
Sharing across instances won't be possible with the next release, but we are going to work on that next as stated in our docs.
